packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -769,12 +788,22 @@ class _RangeSliderState extends State<RangeSlider> with TickerProviderStateMixin
       result = Padding(padding: padding, child: result);
     }
 
-    return FocusableActionDetector(
-      enabled: _enabled,
-      onShowHoverHighlight: _handleHoverChanged,
-      includeFocusSemantics: false,
-      mouseCursor: effectiveMouseCursor,
-      child: result,
+    return Stack(
+      children: <Widget>[
+        //Adds two invisible focus nodes to the range slider for its two thumbs.
+        Row(
+          children: <Widget>[
+            Focus(focusNode: startFocusNode, child: const SizedBox.shrink()),
+            Focus(focusNode: endFocusNode, child: const SizedBox.shrink()),
```

Comment:
This might be good enough for keyboard-based focus traversal. However, this may not be enough for mouse-based focus traversal. On the web it is expected that if I click on a thumb it acquires focus. So if I use the left/right arrow keys, the last thumb I clicked on reacts to the key.

One way to fix this is to position and size the children exactly to cover the thumbs, and hook up a click/tap listener that calls `startFocusNode.requestFocus()` or `endFocusNode.requestFocus()`, as appropriate.

That said, we can also treat it as a separate issue, and fix keyboard first, and mouse later.

Reply:
sounds good, i will treat it as a separate issue